uint8[] image
int32[] bearings
float32 rangeStart
float32 rangeEnd
float32 rangeConversion
time timestamp
